src/main/webapp/calendar-script.js

```diff
@@ -36,9 +36,16 @@ function getEvents(calendar) {
   const urlParams = new URLSearchParams(window.location.search);
   const tripKey = urlParams.get('tripKey');
   const tripKeyQuery = (tripKey != null && tripKey != '') ? '?tripKey=' + tripKey : '';
+  var first = true;
 
   fetch('/get-calendar' + tripKeyQuery).then(response => response.json()).then((events) => {
```

Comment:
We briefly talking about adding one last event for return to hotel? Should I do this or I can walk you through it?

Reply:
Was this done on another branch? Or did I accidentally delete it? Either way, you can walk me through it.

Reply:
Upon further discussion, we decided not to add this. 👍